pkg/policy/manager.go

```diff
@@ -145,6 +138,12 @@ func (m *manager) CreatePolicy(ctx context.Context, policy *pb.Policy) (*pb.Poli
 					Name:  policyRelationName,
 				},
 			},
+			{
+				Operation:  esutil.BULK_CREATE,
+				DocumentId: policyCounterId(policyId),
+				// the counter document's version is used to track the current policy version, but the document itself is empty
+				Message: &emptypb.Empty{},
```

Comment:
You don't need to do this in this PR, but it might be a good idea to update our `esutil.BulkRequestItem` struct to include a `Routing` parameter that we could use here.  Since we know that both the policy document and the policy version document will be on the same shard, we could get a performance boost here if this document was also on that shard.

Reply:
That sounds like a good idea, I'll try to get that in a follow up. Opened https://github.com/rode/rode/issues/97 for it.